src/chart/components/volumes/separate-volumes.component.ts

```diff
@@ -48,9 +48,10 @@ export class SeparateVolumesComponent extends ChartBaseElement {
 	 */
 	activateSeparateVolumes() {
 		if (this.paneManager.panes[SeparateVolumesComponent.UUID] === undefined) {
+			const precision = 1;
```

Comment:
I'm not sure if the precision here should be 1, is it possible that the volume of assets would be not integer?
like crypto for example

Reply:
maybe we should use some automatic precision clculator for all the values for volumes?

Reply:
it's interesting, but looks like other platforms shows only integer volumes, no float part (even for BTC)

Reply:
@KirillBobkov I suggest to you to raise a discussion in our product chat

Reply:
I've made precision === 1 because when we enable crosstool, and it formats crosstool value, value becomes bigger than Yaxis. We cannot adjust Yaxis on crosstool hover, because it will make poor performance, so we work on crosstool label formatting